### ISSUE TITLE
Support AIR installers

### ISSUE THREAD

**Issue by fasterthanlime** [itchio/itch#295]
Docs: http://help.adobe.com/en_US/air/redist/WS485a42d56cd19641-70d979a8124ef20a34b-8000.html#WS485a42d56cd19641-70d979a8124ef20a34b-7ffb

Working example (with http://blackmag-c.itch.io/double-kick-heroes):

> C:\Users\amos\AppData\Roaming\itch\bin\elevate.exe 135831.exe -silent -eulaAccepted -allowDownload -location D:\Games\Itch\apps\UNYWMvahD0ZzDLLs

Use [elevate](https://github.com/itchio/elevate) in case air installation is needed

Where the heck do I get ARH and do we need an authorization to download it via ibrew?

**Comment by fasterthanlime** [comment]
Here's where ARH is hiding: https://forums.adobe.com/thread/756660?tstart=0

Example of silent uninstall:

> arh -uninstallAppSilent com.blackmagic-home.DoubleKickHeroes

The appid isn't a windows-specific GUID, one can find it by parsing `install-folder/DoubleKickHeroes/META-INF/AIR/application.xml`

It looks like that will uninstall an air app on Linux & Mac too. However I'm not sure how self-extracting air installers present themselves on mac & linux, sample files welcome.

New plan is to ship arh on dl.itch.ovh, install it on-demand. I've been wanting to install ibrew packages on-demand and it's very small (<100kb)

**Comment by fasterthanlime** [comment]
Sample AIR app manifest:

``` xml
<?xml version="1.0" encoding="utf-8" ?> 
<application xmlns="http://ns.adobe.com/air/application/3.0"> 
    <id>example.HelloWorld</id> 
    <versionNumber>1.0.1</versionNumber> 
    <filename>Hello World</filename> 
    <name>Example Co. AIR Hello World</name> 
     <description> 
        <text xml:lang="en">This is an example.</text> 
        <text xml:lang="fr">C'est un exemple.</text> 
        <text xml:lang="es">Esto es un ejemplo.</text> 
    </description> 
    <copyright>Copyright (c) 2010 Example Co.</copyright> 
    <initialWindow> 
        <title>Hello World</title> 
        <content> 
            HelloWorld.swf 
        </content> 
    </initialWindow>  
    <icon> 
        <image16x16>icons/smallIcon.png</image16x16> 
        <image32x32>icons/mediumIcon.png</image32x32> 
        <image48x48>icons/bigIcon.png</image48x48> 
        <image128x128>icons/biggerIcon.png</image128x128>  
    </icon> 
</application>
```

**Comment by fasterthanlime** [comment]
Closed by a477e1eacc6c06bee4ef56f72fdc1a2e15769d04 & subsequent commits
